mesheryctl/pkg/utils/helpers.go

```diff
@@ -537,6 +538,15 @@ func PrintToTableWithFooter(header []string, data [][]string, footer []string) {
 	table.Render() // Render the table
 }
 
+// ClearLine clear last line from output
+func ClearLine() {
+	if runtime.GOOS == "windows" {
+		fmt.Print("\r")
+	} else {
+		fmt.Print("\r\033[K")
```

Comment:
@zakisk please provide reference that lists these ASCII characters as best practice for their respective OS'es. Include the reference as a comment.